packages/SwingSet/src/kernel/kernel.js

```diff
@@ -559,6 +634,15 @@ export default function buildKernel(kernelEndowments) {
       buildVatManager(vatID, name, setup, options);
     }
 
+    // TESTONLY: we condition on vatAdminVatSetup and vatAdminDevSetup because
+    // some kernel tests don't care about vats and devices and so don't
+    // initialize properly. We should fix those tests.
+    if (vatAdminVatSetup) {
+      // instantiate and hookup vatAdmin Vat
```

Comment:
I think there should be a way to build the vatManagerVat and vatManagerDevice by adding things to the config object (or to the list of genesisVats/genesisDevices), rather than manually creating them here. We can retain the `if (vatAdminVatSetup)` check, but since/if we don't need to stash `vatAdminVatMgr` anymore, we can rely on the existing loop below to create the necessary vat.